plyer/platforms/linux/wifi.py

```diff
@@ -4,37 +4,76 @@
         - nmcli (Network Manager command line tool)
             It is found in most of the popular distros. Support for other
             managers is not provided yet.
-
-        - python-wifi module
-            `https://wifi.readthedocs.io/en/latest/`
-            `https://github.com/rockymeza/wifi`
-
+    One more thing: Betalistish!!
```

Comment:
Did you mean to include this?

Reply:
Only for beta testers! There are some jobs left to do. But beta testers are needed right now!